darwin/datatypes.py

```diff
@@ -0,0 +1,58 @@
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List, Optional, Set
+
+
+@dataclass(frozen=True, eq=True)
+class AnnotationClass:
+    name: str
+    annotation_type: str
+
+
+@dataclass(frozen=True, eq=True)
+class SubAnnotation:
+    annotation_type: str
+    data: any
+
+
+@dataclass(frozen=True, eq=True)
+class Annotation:
+    annotation_class: AnnotationClass
+    data: any
```

Comment:
Is this actually the right typing?
https://docs.python.org/3/library/typing.html#the-any-type